src/components/standalone/hotspot/StatusContent.vue

```diff
@@ -0,0 +1,212 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import {
+  NeButton,
+  NeSkeleton,
+  NeEmptyState,
+  NeInlineNotification,
+  getAxiosErrorMessage
+} from '@nethserver/vue-tailwind-lib'
+import { faCircleCheck, faCircleXmark } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ubusCall } from '@/lib/standalone/ubus'
+import NeTable from '@/components/standalone/NeTable.vue'
+
+const { t } = useI18n()
+const emit = defineEmits(['goToSetting'])
+
+let hotspotSession: any = ref({})
+let loading = ref(true)
+let activeConfiguration = ref(false)
+let error = ref({
+  notificationTitle: '',
+  notificationDescription: ''
+})
+
+onMounted(async () => {
+  await getConfiguration()
+  await loadListSessions()
+  loading.value = false
+})
+
+async function getConfiguration() {
+  try {
+    let getDataConfiguration = await ubusCall('ns.dedalo', 'get-configuration', {})
+    if (
+      getDataConfiguration &&
+      getDataConfiguration.data &&
+      getDataConfiguration.data.configuration &&
+      getDataConfiguration.data.configuration.connected
+    )
+      activeConfiguration.value = getDataConfiguration.data.configuration.hotspot_id != ''
+  } catch (exception: any) {
+    error.value.notificationTitle = t('error.cannot_retrieve_configuration')
+    error.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  }
+}
+
+async function loadListSessions() {
+  try {
+    const res = await ubusCall('ns.dedalo', 'list-sessions')
+    if (res.data && res.data.sessions && res.data.sessions.length)
+      hotspotSession.value = res.data.sessions
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_load_hotspot_configuration')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+  }
+}
+
+function secondsToHMS(seconds: number): string {
+  if (seconds) {
+    const hours = Math.floor(seconds / 3600)
+    const minutes = Math.floor((seconds % 3600) / 60)
+    const remainingSeconds = seconds % 60
+
+    const formattedHours = hours.toString().padStart(2, '0')
+    const formattedMinutes = minutes.toString().padStart(2, '0')
+    const formattedSeconds = remainingSeconds.toString().padStart(2, '0')
+
+    return `${formattedHours}:${formattedMinutes}:${formattedSeconds}`
+  } else return '-'
+}
+
+function autoConvertSize(sizeInBytes: number): string {
```

Comment:
This function already exists inside `vue-tailwind-lib` and it's called `byteFormat1024`